ptpython/python_input.py

```diff
@@ -980,7 +994,7 @@ def enter_history(self) -> None:
         """
         Display the history.
         """
-        app = get_app()
```

Comment:
I *think* this change is not necessary. ``get_app()`` should always give the running app, taking from the context.